manifests/kustomize/overlays/postgres/kustomization.yaml

```
@@ -39,7 +39,7 @@ vars:
 - name: POSTGRES_PORT
   objref:
     kind: ConfigMap
-    name: model-registry-db-parameters
+    name: metadata-postgres-db-parameters
```

```suggestion
    name: metadata-registry-db-parameters
```
Let's keep it agnostic of the DB type if we can since we can't use multiple DBs and types at the same time

---

I pushed another commit to rename the configmap and secret.